.github/workflows/run-tests.yml

```diff
@@ -20,15 +20,18 @@ jobs:
       max-parallel: 1
       matrix:
         go:
-          - 1.18
-          - 1.19
+          - "1.19"
+          - "1.20"
         clickhouse:
-          - 22.3
-          - 22.8
-          - 22.9
-          - '22.10'
-          - 22.11
-          - latest
+          - "22.3"
+          - "22.8"
+          - "22.9"
```

Comment:
22.9 - 22.11 aren't [maintained](https://github.com/ClickHouse/ClickHouse/blob/master/SECURITY.md) anymore, we can skip them in the tests

Reply:
Thanks. Added a reference.